pyvlx/pyvlx.py

```diff
@@ -37,20 +37,21 @@ def __init__(
         self.loop = loop or asyncio.get_event_loop()
         self.config = Config(self, path, host, password)
         self.connection = Connection(loop=self.loop, config=self.config)
+        self.connection.register_connection_closed_cb(self.on_connection_closed_cb)
         self.heartbeat = Heartbeat(
             pyvlx=self,
             interval=heartbeat_interval,
             load_all_states=heartbeat_load_all_states,
         )
         self.node_updater = NodeUpdater(pyvlx=self)
-        self.connection.register_frame_received_cb(self.node_updater.process_frame)
         self.nodes = Nodes(self)
+        self.connection.register_frame_received_cb(self.node_updater.process_frame)
         self.scenes = Scenes(self)
         self.version = None
         self.protocol_version = None
         self.klf200 = Klf200Gateway(pyvlx=self)
         self.api_call_semaphore = asyncio.Semaphore(1)  # Limit parallel commands
-        PYVLXLOG.debug("Loadig pyvlx v0.2.21")
+        PYVLXLOG.debug("Loadig pyvlx v0.1.76")
```

Comment:
Just a side note: we should remove the version from this log message, as this is something which i will always forget to update when bumping a version.